doc/Advanced.md

```diff
@@ -548,6 +548,24 @@ Below the full QC Post-processing (PP) configuration structure is described. Not
 }
 ```
 
+### External tasks configuration
+
+Below the external task configuration structure is described. Note that more than one external task might be declared inside in the "externalTasks" path.
+
+```json
+{
+  "qc": {
+    "externalTasks": {
+      "External-1": {                       "": "Name of the taks",
+        "active": "true",                   "": "Activation flag. If not \"true\", the Task will not be created.",
+        "detectorName": "TST",              "": "3-letter code of the detector.",
```

Comment:
Are these two actually taken into account?

Reply:
Good question. 
active -> InfrastructureGenerator.cxx: 359
detectorName -> This is indeed happily ignored. The origin is used instead. I propose to leave it like that because it is not so trivial to get the detectorName at the moment we create the MO. I will log an issue in JIRA to remember to do it in a next iteration.